src/app/hm-admin/helper/urlHelper.ts

```diff
@@ -0,0 +1,58 @@
+export const urlHelper = {
+
+  /**
+   * get path with params
+   *
+   * @param {string} url
+   * @param {string} params
+   *
+   * @returns {string}
+   */
+  getPath(url: string, addiionnalParams: any): string {
+    const [path, ] = url.split('?');
+
+    return path + urlHelper.getParams(url, addiionnalParams);
+  },
+
+  /**
+   * transform parameter's object to URL parameters
+   *
+   * @param {string} url
+   * @param {string} params
+   *
+   * @returns {string}
+   */
+  getParams(url: string, params: any): string {
```

Comment:
Regarding this, I'm not sure if we want it

| Feature	Chrome | Firefox (Gecko) | Internet Explorer | Opera | Safari |
| --- | --- | --- | --- | --- |
| Basic support | 49 | 29.0 (29.0) | No support | 36 | No support |
| entries(), keys(), values(), and support of for...of | 49 | 44.0 (44.0) | No support | 36 | No support |